src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -82,6 +85,58 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
             }
         }
 
+        /// <summary>
+        /// Endpoint for retrieving delegated rules between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpPost]
+        [Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
+        [Route("authorization/api/v1/[controller]/GetRules")]
+        public async Task<ActionResult<List<Rule>>> GetRules([FromBody] RuleQuery ruleQuery, [FromQuery] bool onlyDirectDelegations = false)
+        {
+            List<int> coveredByPartyIds = new List<int>();
+            List<int> coveredByUserIds = new List<int>();
+            List<int> offeredByPartyIds = new List<int>();
+            List<string> orgApps = new List<string>();
```

Comment:
orgApps is repeated quite a few times in the code. The list is populated with appIds, to make it more intuitive for other 3.0 developers, you could consider exchanging the term orgApps with appIds.

Reply:
Replaced OrgApp and OrgApps with appId and appIds across the project.